Lib/test/test_xmlrpc.py

```diff
@@ -343,6 +343,97 @@ def run_server():
             self.assertEqual(p.method(), 5)
             self.assertEqual(p.method(), 5)
 
+
+class SimpleXMLRPCDispatcherTestCase(unittest.TestCase):
+    class DispatchExc(Exception):
+        """Raised inside the dispatched functions when checking for
+        chained exceptions"""
+
+    def test_call_registered_func(self):
+        """Calls explicitly registered function
```

Comment:
Comments and docstrings that consist of multiple sentences should have periods at the end of sentences.

If you want the first line of the docstring be without a period, turn the rest of the docstring into a comment.
```python
        """Calls explicitly registered function"""
        # Makes sure any exception raised inside the function has no other
        # exception chained to it
```